lib/fetch/formdata.js

```diff
@@ -94,9 +101,14 @@ class FormData {
     // then return the empty list.
     // 2. Return the values of all entries whose name is name, in order,
     // from this’s entry list.
-    return this[kState]
-      .filter((entry) => entry.name === name)
-      .map((entry) => entry.value)
+    const state = this[kState]
+    const result = []
+    for (let i = 0; i < state.length; ++i) {
+      if (state[i].name === name) {
```

Comment:
what if you extract `state[i]` into something like `const currentState = state[i]` to avoid accessing `state[i]` multiple times?

Reply:
IMO This can only make sense if the array is accessed three or more times. This is because it creates the overhead of variable assignment.
V8 array accesses are already fast and unnecessary.